src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs

```diff
@@ -2149,7 +2149,8 @@ private void CheckInternalMembers(NamedTypeSymbol type, bool isFromSource)
                 Assert.Null(member);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/6190"), WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [Fact]
         public void EmitWithNoResourcesAllPlatforms()
```

Comment:
It looks like sometimes this test is failing, I was hitting that failure too, but, I think, I was able to modify it to make it robust. See changes in #9870.